packages/ui/src/Banner.tsx

```diff
@@ -16,8 +17,16 @@ export const hideBanner = (id: string): Promise<void> => {
   return Unifier.storage.setItem(getKey(id), "true");
 };
 
+export const BannerType = {
+  ACTION: "action",
```

Comment:
What's the benefit of this over ensuring we're using typing properly?

Reply:
yep, I just don't like to type strings. Lemme know if you want me to remove

Reply:
Gotcha! I'm fine with this PR, but we should collectively decide on types vs these const/enums so we can have consistency.